xdp-loader/README.org

```diff
@@ -48,8 +48,13 @@ The supported options are:
 ** -m, --mode <mode>
 Specifies which mode to load the XDP program to be loaded in. The valid values
 are 'native', which is the default in-driver XDP mode, 'skb', which causes the
-so-called /skb mode/ (also known as /generic XDP/) to be used, or 'hw' which
-causes the program to be offloaded to the hardware.
+so-called /skb mode/ (also known as /generic XDP/) to be used, 'hw' which
+causes the program to be offloaded to the hardware, or 'unspecified' which
+leaves it up to the kernel to pick a mode (which it will do by picking native
+mode if the driver supports it, or generic mode otherwise). The default is
+'native'; this is a change from the previous default of 'unspecified', which
```

Comment:
I think you should remove the "; this is a change from...." part, as it makes no sense to keep this history in the documentation file. Maybe we should add a NEWS or RELEASE file to keep track of new features/changes?

Reply:
Yeah, good point, I'll remove the 'previous' bit and just note the difficulty with unspecified without any history.